internal/cortex/querier/queryrange/queryrange.proto

```diff
@@ -98,18 +98,39 @@ message PrometheusResponseQueryableSamplesStatsPerStep {
 message SampleStream {
   repeated cortexpb.LabelPair labels = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "metric", (gogoproto.customtype) = "github.com/thanos-io/thanos/internal/cortex/cortexpb.LabelAdapter"];
   repeated cortexpb.Sample samples = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "values"];
+  repeated SampleHistogramPair histograms = 3 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "histogram"];
```

Comment:
I am not very familiar with the new histogram, so for a sample stream, we will only have either samples or histograms? Can we have them both the same time?

Reply:
From my understanding and according to the Prometheus common code a SampleStream can have both samples and histograms: https://github.com/prometheus/common/blob/4827ac3e6ff608c6959b8a53665e715904297a69/model/value.go#L195-L226

Reply:
Correct, it is possible for a PromQL query to return both histograms and floats. Something like `rate` will do this because it is applied independently on each series.